tests/test_command.py

```diff
@@ -96,7 +97,7 @@ def _eq_cmd_output(self, buf, expected, env_token=False, currents=()):
             assert_lines.insert(0, "environment included OK")
 
         eq_(
-            buf.getvalue().decode("ascii", "replace").strip(),
+            buf.getvalue().decode("ascii", "replace").strip().replace('\r', ''),
```

Comment:
this looks like more windows compatibility stuff.    needs to be separate issue, separate patch.   if you require using windows to develop then we should get a full windows-compatibility feature up and committed first.